src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -81,8 +81,18 @@ public static Picture getPicture() {
         return Collections.emptyList();
     }
 
+    public static List<Event> getMeetingList(Event... meetings) {
+        return Arrays.stream(meetings)
+                .collect(Collectors.toList());
+    }
+
     public static List<Event> getDateList() {
         return Collections.emptyList();
     }
```

Comment:
Should be okay to remove the old `getMeetingList()` and `getDateList()` which returns empty lists right?

Reply:
Nope, this is still used when the `Person` being initialised has no meetings and no dates.